src/main/java/seedu/address/model/delivery/Id.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.model.delivery;
+
+public class Id {
+
+    public static final String MESSAGE_CONSTRAINTS = "Id should be integer";
+    private static int count = 0;
+
+    public String value;
+
+    public Id() {
+        this.value = String.valueOf(Id.count);
+        Id.count += 1;
```

Comment:
Perhaps ID could be unique but also randomly generated instead of being a counter? Since it is more secure.